code/controllers/subsystem/ticker.dm

```diff
@@ -207,31 +207,38 @@ var/datum/subsystem/ticker/ticker
 
 	return 1
 
+/datum/subsystem/ticker/proc/station_explosion_detonation(atom/bomb)
+	if(bomb)	//BOOM
+		var/turf/epi = bomb.loc
+		qdel(bomb)
+		explosion(epi, 255, 255, 255, 255, TRUE, TRUE, 0, TRUE)
```

Comment:
That would actually be MORE laggy. Plus it doesn't even get halfway done before banging donk